setup.py

```diff
@@ -47,15 +47,16 @@
         "twine==3.8.0",  # Package upload tool
     ],
     "dev": [
-        "commitizen>=2.19,<2.20",  # Manage commits and publishing releases
+        # commitizen: Manage commits and publishing releases
+        (here / "cz-requirement.txt").read_text().strip(),
```

Comment:
@fubuloubu Now, `setup.py` and the actions that use CZ share the same source of truth.